serverless/src/exportSearchClean/__tests__/handler.test.js

```diff
@@ -0,0 +1,76 @@
+import knex from 'knex'
+import mockKnex from 'mock-knex'
+import MockDate from 'mockdate'
+
+import * as getDbConnection from '../../util/database/getDbConnection'
+
+import exportSearchClean from '../handler'
+
+const OLD_ENV = process.env
+
+const MOCK_DATE = '2023-04-28T00:00:00.000Z'
+
+let dbTracker
+
+beforeEach(() => {
+  jest.clearAllMocks()
+
+  jest.spyOn(getDbConnection, 'getDbConnection').mockImplementation(() => {
+    const dbCon = knex({
+      client: 'pg',
+      debug: false
+    })
+
+    // Mock the db connection
+    mockKnex.mock(dbCon)
+
+    return dbCon
+  })
+
+  dbTracker = mockKnex.getTracker()
+  dbTracker.install()
+
+  // Manage resetting ENV variables
+  // TODO: This is causing problems with mocking knex but is noted as important for managing process.env
+  // jest.resetModules()
+  process.env = { ...OLD_ENV }
+  delete process.env.NODE_ENV
+
+  // MockDate is used here to overwrite the js Date object. This allows us to
+  // mock changes needed to test the moment functions
+  MockDate.set(MOCK_DATE)
+})
+
+afterEach(async () => {
+  jest.clearAllMocks()
+
+  // Restore any ENV variables overwritten in tests
+  process.env = OLD_ENV
+
+  // reset hacks on built-ins
+  MockDate.reset()
+
+  dbTracker.uninstall()
+})
+
+afterAll(() => {
```

Comment:
This `afterAll` is unnecessary, it is doing the same thing `afterEach` is doing

Reply:
removed afterAll with https://github.com/nasa/earthdata-search/pull/1630/commits/8fd9e7c1ff6ed890af18d3c77f9aa6e99473e5f8